### PR TITLE
Monster Transform Update:

### PR DESCRIPTION
**Pull request by Jedzkie**
Removal of Hard coded showscript command for Monster Transform.
Corrected the MTF_MATK, it should only give 25 MATK instead of 25% MATK

### CODE REVIEW

File: db/re/item_db.conf [hunk 1]
```diff
@@ -83952,6 +83966,7 @@ item_db: (
 	}
 	Script: <"
 		specialeffect2 EF_CLOAKING;
+		showscript "Trans-Form-!! Golem Fo-rm!!";
```
Comment (dastgir) [inline db/re/item_db.conf]: - is intended between Form?

Comment (Jedzkie) [inline db/re/item_db.conf]: Yes.

// 12664 - Trans_Scroll_Golem
item Trans_Scroll_Golem
event OnConsume:
	ShowEffect "" EF_CLOAKING
	ShowScript "" "¨¬?<b>-</b>¨öA-!! ¡Æi¡¤¨ö ¨¡¡À<b>-</b>¢¯E!!"
	Montransform GOLEM 20 "EFST_MTF_MLEATKED"
return

File: src/map/status.c [hunk 2]
```diff
@@ -4952,7 +4952,7 @@ unsigned short status_calc_matk(struct block_list *bl, struct status_change *sc,
 	if (sc->data[SC_MOONLIT_SERENADE])
 		matk += matk * sc->data[SC_MOONLIT_SERENADE]->val2 / 100;
 	if (sc->data[SC_MTF_MATK])
-		matk += matk * sc->data[SC_MTF_MATK]->val1 / 100;
```
Comment (dastgir) [inline src/map/status.c]: Seperate this one into other commit?

File: src/map/status.h [hunk 3]
```diff
@@ -1641,10 +1641,10 @@ enum si_type {
...
 	//SI_ALL_LIGHTGUARD                      = 791,
 	//SI_ALL_LIGHTGUARD_COOL_TIME            = 792,
-	//SI_MTF_MHP                             = 793,
-	//SI_MTF_MSP                             = 794,
-	//SI_MTF_PUMPKIN                         = 795,
-	//SI_MTF_HITFLEE                         = 796,
+	SI_MTF_MHP                               = 793,
+	SI_MTF_MSP                               = 794,
+	SI_MTF_PUMPKIN                           = 795,
+	SI_MTF_HITFLEE                           = 796,
```
Comment (dastgir) [inline src/map/status.h]: Seperate this one too..

File: src/map/status.c [hunk 4]
```diff
@@ -5802,7 +5802,7 @@ short status_calc_fix_aspd(struct block_list *bl, struct status_change *sc, int
 	|| sc->data[SC_WILD_STORM_OPTION]))
 		aspd -= 50; // +5 ASPD
 	if (sc->data[SC_FIGHTINGSPIRIT] && sc->data[SC_FIGHTINGSPIRIT]->val2)
-		aspd -= (bl->type==BL_PC?pc->checkskill((TBL_PC *)bl, RK_RUNEMASTERY):10) / 10 * 40;
+		aspd -= (bl->type == BL_PC ? pc->checkskill((TBL_PC *)bl, RK_RUNEMASTERY) : 10) / 10 * 40;
```
Comment (dastgir) [inline src/map/status.c]: Unrelated whitespace edit o_O

**Comment by Jedzkie** [comment]
@dastgir done.
